zmon_worker_monitor/zmon_worker/tasks/main.py

```diff
@@ -1165,6 +1165,9 @@ def _build_check_context(self, req):
             'redis_host': self.get_redis_host(),
             'redis_port': self.get_redis_port(),
             'zmon_url': MainTask._zmon_url,
+            'stack_name': entity.get('stack_name'),
```

Comment:
let's not treat arbitrary entity properties (such as "stack_name") in any special way...
I would not inject them directly, users can use `entity` as needed.